### PR TITLE
Bring our information about working groups up-to-date also including & respecting the changes discussed for capacity allocation

### PR DESCRIPTION
**Pull request by MarkusTiede**
# Proposed Changes

  - [x] translate AG charter to >> WG charter
  - [x] switch to BalMATE
  - [x] respect capacity allocation meeting discussion result(s) / flow adjustments
  - [x] add reference / information / link to "lean startup cycle" 
  - [x] add reference / information / link to "business request process"

### CODE REVIEW

File: docs/concepts/lifecycle.puml [hunk 1]
```diff
@@ -1,7 +1,86 @@
...
+      * the topic has <b>severity</b> via votes
+      ====
+      <b>OnePager</b> from owner 
+      about high-level <b>scope and goal</b>
+      as <b>Epic in JIRA</b>
+    end note
+
+    partition #LightGrey "Handle new Business Request" {
+        #LightGrey:Create Request;
+        #LightGrey:Community Voting;
```
Comment (joachimprinzbach) [inline docs/concepts/lifecycle.puml:20]: The notes on the right seem to apply for "Create Request" only. I am missing examples / clarification for Voting and Triage & Assign

File: docs/concepts/lifecycle.puml [hunk 2]
```diff
@@ -1,7 +1,86 @@
...
-:Active;
-:Closed;
-stop
+partition #B0E3E6 "Funnel" {
+
+    note right
+      * there is an <b>owner</b>
+      * there is a <b>lead</b> e.g. proposed via owner, BalMATE, ...
+      * the topic has <b>severity</b> via votes
+      ====
```
Comment (joachimprinzbach) [inline docs/concepts/lifecycle.puml:11]: what is the separation for?

File: docs/concepts/lifecycle.puml [hunk 3]
```diff
@@ -1,7 +1,86 @@
...
+        repeat while (DoR fulfilled?)
+    }
+}
+
+partition #BAC8D3 "Ready" {
+    note right
+      the topic has priority
+    end note
+
+    #LightGrey::Prioritize Working Group;
```
Comment (joachimprinzbach) [inline docs/concepts/lifecycle.puml]: remove one colon

**Comment by joachimprinzbach** [comment]
Great approach, I really like trying out the "docs as code" pattern, well done! :)

File: docs/concepts/wg.md [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
-- Grundsätzlich gilt bei Entscheidungen das Mehrheitsprinzip (absolutes Mehr) wobei der Lead das Recht des Stichentscheids hat. In begründeten Fällen kann von diesem Modus abgewichen werden.
-- Besteht das Bedürfnis zur Eskalation ist der CCO der erste Ansprechpartner.
-- Am Ende einer Arbeitsgruppe wird das Resultat dem Leadgremium vorgestellt und von diesem bewertet.
-- Resultat und Bewertung des Leadgremiums werden im folgenden Leitungsprotokoll publiziert.
+## Process organization
+ - Each working group maintains a Confluence page on which the lead, goal / task, dates, participants, (interim) results, decisions and evaluations of the lead committee are logged. 
+- Workgroups have the status **planned** (with start date), **active** (with end date) and **closed**. The status can be seen on the Confluence page.
+- 1-2 weeks before the start of a working group, it will be published in the next [IT management protocol](https://confluence.baloisenet.com/atlassian/x/WYHbLw). Interested parties can then contact the lead.
```
Comment (joachimprinzbach) [inline docs/concepts/wg.md]: Do we want to keep confluence links in public GitHub repos?

**Comment by joachimprinzbach** [comment]
I would consider using a theme, maybe a built-in one: https://plantuml.com/de/theme

File: docs/concepts/lifecycle.puml (hunk 1 above)
Comment (MarkusTiede) [reply docs/concepts/lifecycle.puml:20]: I agree - from our discussions so far we agreed (imho) on taking this as a given input, which is also why 
 - the process starts within refine
 - all boxes in handle new business request are #LightGrey
 - the examples cover what we've discussed to far

File: docs/concepts/lifecycle.puml (hunk 2 above)
Comment (MarkusTiede) [reply docs/concepts/lifecycle.puml:11]: I use the separator to visually split individual and independent (sequential) actions

File: docs/concepts/lifecycle.puml (hunk 3 above)
Comment (MarkusTiede) [reply docs/concepts/lifecycle.puml]: done dada1eb7641590495c35ab346b897d27a58dd2b7

File: docs/concepts/wg.md (hunk 4 above)
Comment (MarkusTiede) [reply docs/concepts/wg.md]: Good question: (imho) due to the fact that we publish confluence, bitbucket and JIRA plugins as open-source for years now it's not a secret using these tools. In addition to that using the unique permalink version does not expose any additional information other than that. Due to that I'd say : yes.

**Comment by MarkusTiede** [comment]
> I would consider using a theme, maybe a built-in one: https://plantuml.com/de/theme

done with 775908c45485f3b66cf144e6b022293558da374a however this slighty breaks explicitly used colors.

**Comment by MarkusTiede** [comment]
- [x] clear mapping + template for instantiate
- [x] alter / extend written documentation
- [x] add touchpoints / embedded timing in SAFe PI / architectural runway heartbeat

**Review by joachimprinzbach — APPROVED** [review]

**Pull request merged**